com.unity.shadergraph/Editor/Generation/Processors/Generator.cs

```diff
@@ -219,7 +310,6 @@ void GenerateShaderPass(int targetIndex, PassDescriptor pass, ActiveFields activ
             // Setup
 
             // Initiailize Collectors
```

Comment:
Here we remove the code to build the per-pass PropertyCollector, as we are using the subShaderProperties instead.